File: mainEx1.c
#include <stdio.h>
#include "ex1.h"

int main(){
    int array1[] = {1,2,3,4,5};
    int array2[] = {10,20,30,40,50};
    //Test games
    printf("Exercise 1\n");
    printf("Sum array1 : %d\n", array_sum(array1, 5));
    printf("Sum array2 : %d\n", array_sum(array2, 5));
    return 0;
}

File: ExercisesInclude/mainEx1.java
package ExercisesInclude;
public class mainEx1 {
    public static void main(String[] args){
        ex1Soluce array1 = new ex1Soluce(new int[]{1, 2, 3, 4, 5});
        ex1Soluce array2 = new ex1Soluce(new int[]{10,20,30,40,50});
        System.out.println("Java : Exercise 1");
        System.out.println("Sum array1 : "  + array1.array_sum());
        System.out.println("Sum array2 : " + array2.array_sum());
    }
}
